crates/database/src/orders.rs

```diff
@@ -727,17 +765,20 @@ pub async fn count_limit_orders_with_outdated_fees(
     ex: &mut PgConnection,
     max_fee_timestamp: DateTime<Utc>,
     min_valid_to: i64,
+    quote_unfunded_orders: bool,
```

Comment:
The name `quote_unfunded_orders` may be incorrect; there's no quoting going on at this layer.